.github/workflows/build-pr.yml

```diff
@@ -0,0 +1,26 @@
+name: build-pr
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v1
```

Comment:
can you please bump to v2?

https://github.com/actions/setup-go#v2

Reply:
done